plugins/accuracy.h

```diff
@@ -28,7 +28,7 @@ struct AccuracyConfig : public ProcedureConfig {
     static constexpr const char * name = "classifier.test";
 
     AccuracyConfig()
-          : mode(CM_BOOLEAN), uniqueScoresOnly(false)
+          : mode(CM_BOOLEAN), uniqueScoresOnly(false), accuracyOverN(1)
```

Comment:
Use member initialization, not a constructor for default values (the configuration classes should have no constructor)